platforms/hyperledger-fabric/charts/orderernode/templates/deployment.yaml

```diff
@@ -237,6 +237,11 @@ spec:
         - name: certificates
           mountPath: /var/hyperledger/orderer/crypto
           readOnly: true
+        resources:
+          requests:
+            memory: {{ .Values.config.pod.resources.requests}}
```

Comment:
It will help to introduce CPU resource as well to the chart. There will be spike in CPU utilization when there are more blocks/transactions to be processed by an orderer or the peer node (reason could be because of parallel transaction execution on multiple channels).

Reply:
fixed